### ISSUE TITLE
VLC Http server Error

### ISSUE THREAD

**Issue by Juanpsd1224** [EugeneDae/VLC-Scheduler#16]
Hi EugeneDae,

I'm having problems running the code on my raspberry pi (with python3.7). I'm running VLC Scheduler at every boot, but sometimes the code gives me an error with the http server. The code puts the file in the playlist but the files are not reproduced by vlc player. I tried with default address (127.0.0.1:8080) but isn''t works.

Can you help me?

Regards

![image](https://user-images.githubusercontent.com/46231665/61015284-7ceb0900-a350-11e9-8e0d-533bd70521ea.png)

**Comment by EugeneDae** [comment]
Hi @Juanpsd1224

What happens exactly when you try with the default address?

Do you also run VLC at every boot or just VLC Scheduler?

**Comment by Juanpsd1224** [comment]
The error appears with any address and port... I only run VLC Scheduler at every boot..

Some times when I reboot Raspberry Pi a message appears : ""Found VLC Instance" or something like that

**Comment by EugeneDae** [comment]
Try removing `http` from `extraintf` in vlcscheduler.yaml, reboot and see what happens.

Change:

```
vlc:
    ...
    extraintf: 'http,luaintf'
```

to:

```
vlc:
    ...
    extraintf: 'luaintf'
```

**Comment by EugeneDae** [comment]
Also please post your vlcrc (the config file). Possible location: `/home/pi/.config/vlc/vlcrc`

If not found, use:
`find / -name "vlcrc"`

**Comment by Juanpsd1224** [comment]
Hi,
I changed the codes but the same error appears

[vlcrc.txt](https://github.com/EugeneDae/VLC-Scheduler/files/3384006/vlcrc.txt)

**Comment by EugeneDae** [comment]
Try:

1. Add `http` back to vlcscheduler.yaml, like so:

```
vlc:
    ...
    extraintf: 'http,luaintf'
```

2. In your vlcrc around line 4338 change `extraintf=lua:http` to `extraintf=lua`

3. Reboot.

**Comment by Juanpsd1224** [comment]
It gives same error

![Screenshot_133](https://user-images.githubusercontent.com/46231665/61087601-b462bf80-a3fb-11e9-95c0-0665a92092ae.png)

**Comment by EugeneDae** [comment]
1. Remove again `http` from `extraintf` in vlcscheduler.yaml, don't change anything in vlcrc, and see what happens.

2. Also run `netstat -tulpn` and posts results.

**Comment by Juanpsd1224** [comment]
Hi, 

I removed changes and this happened

Regards

![Screenshot_135](https://user-images.githubusercontent.com/46231665/61124191-e3187e80-a46b-11e9-9103-0fd355e05a82.png)
![Screenshot_137](https://user-images.githubusercontent.com/46231665/61124199-ec095000-a46b-11e9-86ee-1b7c705fc12a.png)

**Comment by Juanpsd1224** [comment]
Hi EugeneDae,

I think that I solved the issue... It was the way that vlc scheduler has been starting at every boot. I was doing with .bashrc file and LXDE terminal autostart config file which starts vlc scheduler script before all raspbian OS services and processes starts.

So I changed the way that the raspberry boots vlc scheduler so that it was after all services and processes has been load and now it's working fine.

Thanks for your help... And excuse me if my english doesn't understands

Regards
